pkg/util/cluster/cluster.go

```diff
@@ -448,21 +448,22 @@ func (c *Cluster) createCluster(ctx context.Context, vnetResourceGroup, clusterN
 		}
 
 		oc.Properties.WorkerProfiles[0].VMSize = api.VMSizeStandardD2sV3
+		oc.Properties.ClusterProfile.FipsValidatedModules = api.FipsValidatedModulesDisabled
```

Comment:
This should be removed.  We can create FIPs clusters anywhere.  

Also, can you set FIPs to enabled by default?  It should be in the api doc above on line 403